themes-default/slim/views/schedule.mako

```diff
@@ -19,6 +19,82 @@ const startVue = () => {
         el: '#vue-wrap',
         data() {
             return {};
+        },
+        mounted() {
+            if ($.isMeta({ layout: 'schedule' }, ['list'])) {
+                const sortCodes = {
+                    date: 0,
+                    show: 2,
```

Comment:
GET http://localhost:8081/js/schedule/index.js?10032 net::ERR_ABORTED
Error on load of index.js.

Reply:
From what I see, the import just needs to be removed from `main.mako`.
https://github.com/pymedusa/Medusa/blob/bca8f0c03547625646faec742988e192855e9d97/themes-default/slim/views/layouts/main.mako#L96

Reply:
Same as other script tag, can be removed.